Website/src/rendering/src/components/SpeakerInfoLine.tsx

```diff
@@ -0,0 +1,14 @@
+import { Field, Text } from '@sitecore-jss/sitecore-jss-nextjs';
+
+export function SpeakerInfoLine(props: { title: string; field: Field<string> }): JSX.Element {
+  if (!props.field) {
+    return <></>;
+  }
+
+  return (
+    <span className="block">
```

Comment:
Can you please move the `block` and `font-bold` Tailwind classes to a CSS file with `@apply` directives?